t/app/controller/waste_peterborough.t

```diff
@@ -1312,7 +1312,6 @@ FixMyStreet::override_config {
             if ($days) {
                 my $email = $mech->get_email->as_string;
                 like $email, qr/26 August/;
-                like $email, qr/Wardrobe/;
```

Comment:
Might be better as `unlike`, that way we're testing the absence of something rather than the absence of a test for something.

```suggestion
                unlike $email, qr/Wardrobe/;
```